src/expandable-section/expandable-section-header.tsx

```diff
@@ -62,7 +74,43 @@ export const ExpandableSectionHeader = ({
         >
           {icon}
         </button>
-        {children}
+        {headerText ?? header}
+      </div>
+    );
+  }
+
+  if (variant === 'container' && headerText) {
```

Comment:
We have 3 branches in this component with very different markup. Any chances to refactor this, for example, into `ExpandableContainerHeader`, `ExpandableNavigationHeader` and `ExpandableDefaultHeader` (for example)?

Reply:
good point, will update in the next revision